kasa/interfaces/lightpreset.py

```diff
@@ -0,0 +1,89 @@
+"""Module for LightPreset base class."""
+
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import Sequence
+
+from ..feature import Feature
+from ..module import Module
+from .light import LightState
+
+
+class LightPresetInfo(ABC):
```

Comment:
`LightPresetData`, perhaps?

Reply:
Ah this shouldn't be here, accidentally left behind after moving to adding a `LightState` in the `Light` module which can be used to return preset state or passing into `light.set_state(`LightState`) in the `Light` module.